Remotion/Development/Web/UnitTesting/UI/Controls/Rendering/HtmlHelperBase.cs

```diff
@@ -192,10 +194,11 @@ public void AssertAttribute (XmlNode node, string attributeName, string attribut
     /// </summary>
     public void AssertAttribute (XmlNode node, string attributeName, string attributeValue, AttributeValueCompareMode mode)
     {
-      XmlAttribute attribute = node.Attributes[attributeName];
+      // TODO: Assert node.Attributes not null
+      XmlAttribute? attribute = node.Attributes![attributeName];
```

Comment:
XmlNode.Attributes can be null? Strange.

Reply:
Indeed: https://source.dot.net/#System.Private.Xml/System/Xml/Dom/XmlNode.cs,177